test/unit/views/ViewAllApplicationsPageSpec.scala

```diff
@@ -75,7 +76,7 @@ class ViewAllApplicationsPageSpec extends UnitSpec with OneServerPerSuite with M
       def shouldViewWithAppShowAlert(environment: String, termsOfUseStatus: TermsOfUseStatus, shouldAlert: Boolean) = {
 
         val appSummaries = Seq(ApplicationSummary("1111", "App name 1", environment, Role.ADMINISTRATOR,
-          termsOfUseStatus, State.TESTING, DateTimeUtils.now))
+          termsOfUseStatus, State.TESTING, DateTimeUtils.now, DateTimeUtils.now))
```

Comment:
Do you mean for these to be the same ?  If so they need to be extracted to a local var or you risk a flakey test

Reply:
Not for this one, but the one on top assumes they'll be the same in the assertion so I've changed it so both values come from the same instance.